websites/P/Polywork/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({
+		clientId: "1021098794894499952",
+	}),
+
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+	presence.on("UpdateData", async () => {
+		const presenceData: PresenceData = {
+				largeImageKey: "https://i.imgur.com/8KLMTMQ.png",
+				details: "Browsing..",
+				startTimestamp: browsingTimestamp,
+			},
+			[pathname, hostname, href, URL] = document.location
```

Comment:
oops u don't use URL
```suggestion
			[ pathname, hostname, href ] = document.location;
```